src/test/scala/org/virtuslab/beholder/form/FromFiltersSeqTest.scala

```diff
@@ -11,11 +11,11 @@ import org.virtuslab.unicorn.{ UnicornPlay, UnicornWrapper }
 import scala.concurrent.ExecutionContext.Implicits.global
 
 class FormSeqTestRepository(override val unicorn: UnicornPlay[Long])
-    extends UserMachinesViewComponent
-    with FormFormatterComponent
-    with FormFilterComponent
-    with FromFilterFieldsComponent
-    with UnicornWrapper[Long] {
```

Comment:
Could you revert unintended formatting please? :)

Reply:
This reformatting is result of bumping scalariform to version 1.8.2. In order to revert it i would need to either downgrade scalariform to 1.3.0 (and sbt to 0.13) or disable formatting at compile.